### ISSUE TITLE
3.3.9 Context Creation WGL Error - Windows 11 on Parallels for Mac VM

### ISSUE THREAD

**Issue by NateIsStalling** [glfw/glfw#2467]
GLFW 3.3.9 context creation fails to initialize in Windows 11 Parallels VM on Mac with the error "WGL: Failed to retrieve pixel format attributes: The operation completed successfully."

Confirmed latest 3.4.0 commit `b4c3ef9d0fdf46845f3e81e5d989dab06e71e6c1` presents the same glfwinfo error as 3.3.9 build (3.3.9 glfwinfo output listed below).

GLFW 3.3.8 appears to initialize successfully with the same Device/VM configuration - 3.3.8 glfwinfo output listed further below.

Issue appears related to issue discussed in https://github.com/glfw/glfw/pull/2191

**Device/VM Info**
OS and version: `Windows 11 Pro, Version 22H2, OS build 22621.2861, 64-bit operating system, x64-based processor`
VM name and version: `Parallels Desktop 19 for Mac Pro Edition, Version 19.2.1 (54832)`
GPU and driver: `Parallels Display Adapter (WDDM), 4.0 GB, Version 19.2.1.54832`

**GLFW 3.3.9 glfwinfo output**
Release or commit: `e2c92645460f680fd272fd2eed591efb2be7dc31`
Version string: `3.3.9 Win32 WGL EGL OSMesa VisualC`
glfwinfo output:
```
glfw\build-static\tests\Debug>glfwinfo
GLFW header version: 3.3.9
GLFW library version: 3.3.9
GLFW library version string: "3.3.9 Win32 WGL EGL OSMesa VisualC"
Error: WGL: Failed to retrieve pixel format attributes: The operation completed successfully.
```

**GLFW 3.3.8 glfwinfo output**
Release or commit: `7482de6071d21db77a7236155da44c172a7f6c9e`
Version string: `3.3.8 Win32 WGL EGL OSMesa VisualC`
glfwinfo output:
```
glfw\build-static\tests\Debug>glfwinfo
GLFW header version: 3.3.8
GLFW library version: 3.3.8
GLFW library version string: "3.3.8 Win32 WGL EGL OSMesa VisualC"
OpenGL context version string: "4.1 ATI-5.2.4"
OpenGL context version parsed by GLFW: 4.1.0
OpenGL context flags (0x00000000):
OpenGL context flags parsed by GLFW:
OpenGL profile mask (0x00000002): compat
OpenGL profile mask parsed by GLFW: compat
OpenGL context renderer string: "Parallels using AMD Radeon Pro 5500M OpenGL Engine (Compat)"
OpenGL context vendor string: "Parallels and ATI Technologies Inc."
OpenGL context shading language version: "4.10"
OpenGL framebuffer:
 red: 8 green: 8 blue: 8 alpha: 8 depth: 24 stencil: 8
 samples: 0 sample buffers: 0
 accum red: 16 accum green: 16 accum blue: 16 accum alpha: 16 aux buffers: 0
Vulkan loader: missing
```

**Comment by dougbinks** [comment]
Many thanks for the information in your issue report. This does indeed look like #2191

**Comment by elmindreda** [comment]
This should be fixed now with 00e86d4b733103a23278fe53ce58a0d14dd47d32. Thank you again for the bug report. Please let us know if you would like your credit line changed.
